Modules/Quotas/src/Quotas.Application/DTOs/MetricDTO.cs

```diff
@@ -14,8 +14,8 @@ public MetricDTO(Metric metric)
         DisplayName = metric.DisplayName;
     }
 
-    public string Key { get; set; }
-    public string DisplayName { get; set; }
+    public string Key { get; set; } = null!;
```

Comment:
This is fine as it is, however...

This class is not persisted so I think the empty constructor is not used by the EF Core, and otherwise it has zero references in the project. I tried deleting it - it compiles, but I haven't run the tests. I think it makes sense to try deleting it and then running the tests in the pipeline, and if that works then these `null!` initializers can be dropped as well.

Reply:
I agree.

Reply:
Unfortunately, we were wrong :) CNS transport tests fail, this is the reason as stated in the logs: `"Message":"Cannot dynamically create an instance of type 'Backbone.Modules.Quotas.Application.DTOs.MetricDTO'. Reason: No parameterless constructor defined."`

So it seems the that constructor has to stay...